flexget/plugins/clients/qbittorrent.py

```diff
@@ -62,31 +66,31 @@ def connect(self, config):
         'localhost'.
         """
         self.session = Session()
-        self.url = 'http://{}:{}'.format(config['host'], config['port'])
+        self.url = '{}://{}:{}'.format('http' if config['use_ssl']==False else 'https', config['host'], config['port'])
         if config.get('username') and config.get('password'):
             data = {'username': config['username'],
                     'password': config['password']}
-            self._request('post', self.url + '/login', data=data, msg_on_fail='Authentication failed.')
+            self._request('post', self.url + '/login', data=data, msg_on_fail='Authentication failed.', verify=config['verify_cert'])
         log.debug('Successfully connected to qBittorrent')
         self.connected = True
 
-    def add_torrent_file(self, file_path, data):
+    def add_torrent_file(self, file_path, data, config):
         if not self.connected:
             raise plugin.PluginError('Not connected.')
         multipart_data = {k: (None, v) for k, v in data.items()}
         with open(file_path, 'rb') as f:
             multipart_data['torrents'] = f
             self._request('post', self.url + '/command/upload', msg_on_fail='Failed to add file to qBittorrent',
                           files=multipart_data)
-        log.debug('Added torrent file %s to qBittorrent', file_path)
+        log.debug('Added torrent file %s to qBittorrent', file_path, verify=config['verify_cert'])
```

Comment:
If you wanna add verify status in here, fix the message as well

Reply:
Thanks for the insights!
This one was a mistake, I wanted to add the parameter to the previous line.
Everything else has been fixed.